extensions/libarchive/CompressContent.h

```diff
@@ -386,7 +386,7 @@ class CompressContent : public core::Processor {
               if (io::isError(writeret) || gsl::narrow<size_t>(writeret) != ret) {
                 return -1;
               }
-              read_size += ret;
+              read_size += gsl::narrow<int64_t>(ret);
```

Comment:
could we change `read_size`'s type to `size_t`, we could get rid of two narrows

Reply:
done in 5c6ba7b